packages/unit-testing/stubs/translation/Handle.js

```diff
@@ -0,0 +1,25 @@
+define([], function() {
```

Comment:
Missing Translator.js object?

Reply:
It goes the other way around, Handle.js is injected into translation.js.